reports: 20190725 Cointime Summit 2019 in Ho Chi Minh City, Vietnam

duyenemdo opened the pull request:
Report for the 3 days of cryptocurrency events in Ho Chi Minh City

duyenemdo committed:
Create 20190726-cointime-summit-2019.mp

duyenemdo committed:
Create 20190728-Vietnamstakingeconomy-vietnam.mp

duyenemdo committed:
Create 20190728-VietnamStakingEconomy-Vietnam

duyenemdo committed:
Delete 20190728-VietnamStakingEconomy-Vietnam

duyenemdo committed:
Update 20190726-cointime-summit-2019.mp

duyenemdo committed:
Update 20190726-cointime-summit-2019.mp

duyenemdo committed:
Update 20190726-cointime-summit-2019.mp

duyenemdo committed:
Delete 20190728-Vietnamstakingeconomy-vietnam.mp

duyenemdo committed:
Add files via upload

duyenemdo committed:
Update 20190726-cointime-summit-2019.mp

duyenemdo committed:
Update 20190726-cointime-summit-2019.mp

duyenemdo committed:
Rename 20190726-cointime-summit-2019.mp to 20190726-cointime-summit-2019.md

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

duyenemdo committed:
images

duyenemdo committed:
Add files via upload

duyenemdo committed:
Delete 2627-2.jpg

duyenemdo committed:
Delete 2627-3.jpg

duyenemdo committed:
Delete 2627.jpg

duyenemdo committed:
Delete 2807-1.jpg

duyenemdo committed:
Delete 2807-2.jpg

duyenemdo committed:
Delete 2807-3.jpg

duyenemdo committed:
Delete 2807-4.jpg

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

duyenemdo committed:
Delete event pictures

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

duyenemdo committed:
Update 20190726-cointime-summit-2019.md

xaur commented:
Hey thanks for the reports!

1. Was it one long running event, two very closely related events or two separate events? Would it be better to split into two reports?
2. Are there any meetup.com/eventbrite/twitter/facebook links for the events?

xaur commented:
3\. Do swag photos belong to first or second event? (awesome swag btw!)

duyenemdo commented:
Dear,

Thanks for your reviewing. About your questions:

1. It was 2 separated events. The first one, Cointime summit 2019, was held
for 2 days (25-26/07) and the second one, Vietnam Staking Economy, was on
28th July.

2. At first, I wrote 2 separated reports but it seemed better to be merged
into one since mostly the ideas in the report were came up after the two
events. In addition, actually we planed to join only the first event.
However, we made some contacts there, then they invited us to join theirs.

3. The swag photos belong to both events.

If there are any questions, please feel free to ask.

Regards,

Em.


On Sun, Aug 4, 2019 at 22:21 xaur <notifications@github.com> wrote:

> 3. Do swag photos belong to first or second event? (awesome swag btw!)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/decredcommunity/events/pull/7?email_source=notifications&email_token=ALATJQOQLFSGZ5B34MDOYS3QC3XYNA5CNFSM4IJCDX62YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD3QD3HQ#issuecomment-518012318>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALATJQNGS5OVVOQ3HESLHJDQC3XYNANCNFSM4IJCDX6Q>
> .
>
-- 
Được gửi từ Ipad3

xaur committed:
Group photos

xaur commented:
I try to keep the repository text only and avoid comitting photos because it grows size quickly. Need to host them elsewhere.

xaur commented:
Let's try to store some in the comment.

![photo_2019-08-02_11-55-14](https://user-images.githubusercontent.com/24484727/62710059-1fe18200-b9e6-11e9-9eaa-06bb1a8be921.jpg)
![photo_2019-08-02_11-55-22](https://user-images.githubusercontent.com/24484727/62710061-1fe18200-b9e6-11e9-83a0-df234e50c996.jpg)
![photo_2019-08-02_11-55-26](https://user-images.githubusercontent.com/24484727/62710063-1fe18200-b9e6-11e9-958a-0764d4678510.jpg)
![photo_2019-08-02_11-57-47](https://user-images.githubusercontent.com/24484727/62710066-207a1880-b9e6-11e9-9838-77c84451db5d.jpg)
![photo_2019-08-02_11-57-50](https://user-images.githubusercontent.com/24484727/62710068-207a1880-b9e6-11e9-861f-bfdaaf4bc951.jpg)

xaur committed:
Use external photo links

xaur commented:
Ok it works. Notice how the photos from Twitter are cached and proxied by GitHub. Swag links to images uploaded in the comment above work too. Not ideal, but will work for now.

xaur committed:
Delete images

xaur committed:
Fix formatting and typos

xaur commented:
Added VN links to [social media](https://github.com/decredcommunity/wiki/blob/master/wiki/social-media.md) wiki page.

xaur commented:
@duyenemdo OK I went through the report, fixed formatting and some typos.

Small remaining issues:

* Are there any event links? For example to Cointime and Staking Economy websites, or meetup.com, eventbrite, announcement tweets, twitter accounts or hashtags? I'd like to add links to Info like [in here](https://github.com/duyenemdo/events/blob/master/reports/20190524-blockchain-ai-digital-assets-conference-lagos-nigeria.md).
* I fixed capitalization of some names (e.g. tomochain > TomoChain) but may have missed some
* This repository is public, please make sure no private/sensitive/strategic information is added (e.g. investor names, or are they ok with it?)

I'm not sure yet about 1 merged report vs 2 reports. Reasons to have a single report: it was one trip/effort, one set of swag, the report contains reflections common for both events. Reasons to split: in 5 places it was necessary to clarify the event (Info, Agenda, Reflection, Contacts, Media). Please correct if I missed something.

I'm not going to force hard rules at this point, rather I'm using this new situation to establish best practices. We can merge it either way.

Please let me know if you plan any more changes or if I can proceed with the merge.

duyenemdo commented:
I added facebook and twitter addresses on the link you gave me already.

Facebook: https://www.facebook.com/Decred-DCR-Vietnam-108991833777572/

Twitter: https://twitter.com/DecredVietnam


And here are the posts:


1. Facebook posts:

Cointime Summit 2019:
https://www.facebook.com/permalink.php?story_fbid=109008563775899&id=108991833777572&__xts__[0]=68.ARDAySMNKjy0CbO2bvKcd4un_IMJDXSgsIWIEnigl1FWGNGr0B9ZakxK6Y8zNfBP6HJYxfGsLTTSsyMliYr_FldEFFn-4Xgdo-UozBhNj9_k3y_XHGiWHyPEbDJz2Op7O-uMetVIcEckyzm0UGyg8wvUZIW8IpRwRyRJXJrGvOiSwN0klaVNUQaFFi7SpmwuAMFl_R9uhOVVpg6gHTfHSeh2sJZltm2x7BaaNOjBY97MH1sOe0jiwh8Nnqe63cQWI161k1qd2x1NpFRHvxzrT4lm5T3BW6nTDAA5W8nZdrtRfr_AdAfR9ix-1kQVBBJ5fragf0mgfiRBA_H_kejFDos&__tn__=-R



Staking Economy Vietnam:
https://www.facebook.com/permalink.php?story_fbid=109014860441936&id=108991833777572&__xts__[0]=68.ARC6qJqzpEygBI0e7goerQMhCji2seO_5b6viY28WfON1Zw9A1QbvO7bRH0PFM2V96GTJNx2z64dvIhaY-pFShPh8IQA97cxoq9Hqqbi7l3IJdMb0C3rKxJd9TtBGoUzuq-uA9gSHon4lJX_bqWBjLVvl1GL9wCxxduuqxq7Z_6KWma96Ojsf8o9_ncsLyWNf5lfNCXA9m_vsH4kMkQkNsmbP2QVhLXHhwifoAW2y5ej32v3TMQrd-XnxrFMoNKFxZLAxkdXtusbMKRNLC9ILVr7vk9yFMbSh0TIFSidlL1Kzh0GQyXHZ2Qmp1lPsCOHLJ5snLoTqiMYZDC_PvffbC4&__tn__=-R


2. Twitter posts:

Cointime Summit 2019:
https://twitter.com/DecredVietnam/status/1157148709236305921

Staking Economy Vietnam:
https://twitter.com/DecredVietnam/status/1157150461947813888


Links of the events:

Cointime Summit 2019: https://www.facebook.com/events/2400009453610808/

Staking Economy Vietnam:
https://www.facebook.com/events/dreamplex-2/vietnam-staking-economy/2251343728284813/

Vào Th 5, 8 thg 8, 2019 vào lúc 22:27 xaur <notifications@github.com> đã
viết:

> @duyenemdo <https://github.com/duyenemdo> OK I went through the report,
> fixed formatting and some typos.
>
> Small remaining issues:
>
>    - Are there any event links? For example to Cointime and Staking
>    Economy websites, or meetup.com, eventbrite, announcement tweets,
>    twitter accounts or hashtags? I'd like to add links to Info like in
>    here
>    <https://github.com/duyenemdo/events/blob/master/reports/20190524-blockchain-ai-digital-assets-conference-lagos-nigeria.md>
>    .
>    - I fixed capitalization of some names (e.g. tomochain > TomoChain)
>    but may have missed some
>    - This repository is public, please make sure no
>    private/sensitive/strategic information is added (e.g. investor names, or
>    are they ok with it?)
>
> I'm not sure yet about 1 merged report vs 2 reports. Reasons to have a
> single report: it was one trip/effort, one set of swag, the report contains
> reflections common for both events. Reasons to split: in 5 places it was
> necessary to clarify the event (Info, Agenda, Reflection, Contacts, Media).
> Please correct if I missed something.
>
> I'm not going to force hard rules at this point, rather I'm using this new
> situation to establish best practices. We can merge it either way.
>
> Please let me know if you plan any more changes or if I can proceed with
> the merge.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/decredcommunity/events/pull/7?email_source=notifications&email_token=ALATJQOHYVQ53Z26W4GN363QDQ3OBA5CNFSM4IJCDX62YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD337PNY#issuecomment-519567287>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALATJQNPU4LB6DDID7BYIRTQDQ3OBANCNFSM4IJCDX6Q>
> .
>

duyenemdo commented:
Thanks a lot for your help!

Vào CN, 11 thg 8, 2019 vào lúc 21:23 Duyên Em Đỗ <duyenem2711998@gmail.com>
đã viết:

> I added facebook and twitter addresses on the link you gave me already.
>
> Facebook: https://www.facebook.com/Decred-DCR-Vietnam-108991833777572/
>
> Twitter: https://twitter.com/DecredVietnam
>
>
> And here are the posts:
>
>
> 1. Facebook post:
>
> Cointime Summit 2019:
> https://www.facebook.com/permalink.php?story_fbid=109008563775899&id=108991833777572&__xts__[0]=68.ARDAySMNKjy0CbO2bvKcd4un_IMJDXSgsIWIEnigl1FWGNGr0B9ZakxK6Y8zNfBP6HJYxfGsLTTSsyMliYr_FldEFFn-4Xgdo-UozBhNj9_k3y_XHGiWHyPEbDJz2Op7O-uMetVIcEckyzm0UGyg8wvUZIW8IpRwRyRJXJrGvOiSwN0klaVNUQaFFi7SpmwuAMFl_R9uhOVVpg6gHTfHSeh2sJZltm2x7BaaNOjBY97MH1sOe0jiwh8Nnqe63cQWI161k1qd2x1NpFRHvxzrT4lm5T3BW6nTDAA5W8nZdrtRfr_AdAfR9ix-1kQVBBJ5fragf0mgfiRBA_H_kejFDos&__tn__=-R
>
>
>
> Staking Economy Vietnam:
> https://www.facebook.com/permalink.php?story_fbid=109014860441936&id=108991833777572&__xts__[0]=68.ARC6qJqzpEygBI0e7goerQMhCji2seO_5b6viY28WfON1Zw9A1QbvO7bRH0PFM2V96GTJNx2z64dvIhaY-pFShPh8IQA97cxoq9Hqqbi7l3IJdMb0C3rKxJd9TtBGoUzuq-uA9gSHon4lJX_bqWBjLVvl1GL9wCxxduuqxq7Z_6KWma96Ojsf8o9_ncsLyWNf5lfNCXA9m_vsH4kMkQkNsmbP2QVhLXHhwifoAW2y5ej32v3TMQrd-XnxrFMoNKFxZLAxkdXtusbMKRNLC9ILVr7vk9yFMbSh0TIFSidlL1Kzh0GQyXHZ2Qmp1lPsCOHLJ5snLoTqiMYZDC_PvffbC4&__tn__=-R
>
>
> 2. Twitter posts:
>
> Cointime Summit 2019:
> https://twitter.com/DecredVietnam/status/1157148709236305921
>
> Staking Economy Vietnam:
> https://twitter.com/DecredVietnam/status/1157150461947813888
>
>
> Links of the events:
>
> Cointime Summit 2019: https://www.facebook.com/events/2400009453610808/
>
> Staking Economy Vietnam:
> https://www.facebook.com/events/dreamplex-2/vietnam-staking-economy/2251343728284813/
>
> Vào Th 5, 8 thg 8, 2019 vào lúc 22:27 xaur <notifications@github.com> đã
> viết:
>
>> @duyenemdo <https://github.com/duyenemdo> OK I went through the report,
>> fixed formatting and some typos.
>>
>> Small remaining issues:
>>
>>    - Are there any event links? For example to Cointime and Staking
>>    Economy websites, or meetup.com, eventbrite, announcement tweets,
>>    twitter accounts or hashtags? I'd like to add links to Info like in
>>    here
>>    <https://github.com/duyenemdo/events/blob/master/reports/20190524-blockchain-ai-digital-assets-conference-lagos-nigeria.md>
>>    .
>>    - I fixed capitalization of some names (e.g. tomochain > TomoChain)
>>    but may have missed some
>>    - This repository is public, please make sure no
>>    private/sensitive/strategic information is added (e.g. investor names, or
>>    are they ok with it?)
>>
>> I'm not sure yet about 1 merged report vs 2 reports. Reasons to have a
>> single report: it was one trip/effort, one set of swag, the report contains
>> reflections common for both events. Reasons to split: in 5 places it was
>> necessary to clarify the event (Info, Agenda, Reflection, Contacts, Media).
>> Please correct if I missed something.
>>
>> I'm not going to force hard rules at this point, rather I'm using this
>> new situation to establish best practices. We can merge it either way.
>>
>> Please let me know if you plan any more changes or if I can proceed with
>> the merge.
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/decredcommunity/events/pull/7?email_source=notifications&email_token=ALATJQOHYVQ53Z26W4GN363QDQ3OBA5CNFSM4IJCDX62YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD337PNY#issuecomment-519567287>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/ALATJQNPU4LB6DDID7BYIRTQDQ3OBANCNFSM4IJCDX6Q>
>> .
>>
>

xaur committed:
Add links, fixes

xaur committed:
Fix odd Because

xaur committed:
Obfuscate contacts

xaur commented:
Thanks for the links @duyenemdo . I added those and fixed a few more bugs. Also reworded odd-looking "Host our own conference." and "Because:" paragraphs in this change:

https://github.com/decredcommunity/events/pull/7/commits/726953ccdbe772a1d4b1c63c343e716e3b68549b

xaur committed:
Rename file

xaur commented:
@duyenemdo @raedah I was really curious how it would look in two different reports and tried it out in my fork:

https://github.com/xaur/events/blob/vn2/reports/20190725-cointime-summit-ho-chi-minh-city-vietnam.md

https://github.com/xaur/events/blob/vn2/reports/20190728-staking-economy-ho-chi-minh-city-vietnam.md

You can see the exact changes in this commit: https://github.com/xaur/events/commit/80fd3d92d20fcd42fbe9360578ffcf486e04602f

This is just a demo, the final word is for you guys. What do you say, merge as one or as two?

xaur commented:
Ok lacking response I'm picking the first option and merging the combined report. Here's the final link for dissemination:

https://github.com/decredcommunity/events/blob/master/reports/20190725-cointime-summit-ho-chi-minh-city-vietnam.md

In case you like the split version, I can add that in another commit and the Cointime link will not break. Just let me know.

And sorry for the long processing of this report. I was pulled by DJ and went back and forth about the two reports in one. Hopefully this will be much quicker next time.

xaur commented:
https://www.reddit.com/r/decred/comments/cqexl6/report_cointime_summit_2019_vietnam_staking/
